collect_app/src/main/java/org/odk/collect/android/utilities/gdrive/GoogleAccountsManager.java

```diff
@@ -89,8 +92,20 @@ private void initCredential(@NonNull Context context) {
                 .usingOAuth2(context, Collections.singletonList(DriveScopes.DRIVE))
                 .setBackOff(new ExponentialBackOff());
 
-        intentChooseAccount = credential.newChooseAccountIntent();
         themeUtils = new ThemeUtils(context);
+
+        if (themeUtils.getAccountPickerTheme() == 1) {
+            intentChooseAccount = AccountManager.newChooseAccountIntent(null,
```

Comment:
Do these two methods definitely do the same thing? I'm a little worried about using a whole different integration based on the theme.

Reply:
Hi @seadowg! Thanks for reviewing. `AccountPicker` is a part of Google Play services. I went through the docs of [`AccountPicker.newChooseAccountIntent()`](https://developers.google.com/android/reference/com/google/android/gms/common/AccountPicker#newChooseAccountIntent(com.google.android.gms.common.AccountPicker.AccountChooserOptions)) and [`AccountManager.newChooseAccountIntent()`](https://developer.android.com/reference/android/accounts/AccountManager#newChooseAccountIntent(android.accounts.Account,%2520java.util.ArrayList%253Candroid.accounts.Account%253E,%2520java.lang.String%255B%255D,%2520boolean,%2520java.lang.String,%2520java.lang.String,%2520java.lang.String%255B%255D,%2520android.os.Bundle)), and found that both take the same arguments, and do the same thing.

Reply:
Using a different integration just to fix the theme, seems a little strange to me. But I searched, and I was not able to change the theme in either of the cases with `AccountPicker` or `AccountManager`, by overriding variables in the intent.

Reply:
Hmmm I'm not sure about the ramifications of this but I'm not really comfortable with it without doing some research. @grzesiek2010 do you have an opinion on this?

Reply:
For me it doesn't work and I also don't understand the trick here. If I go to the body of `newChooseAccountIntent()`
it returns:
`    return AccountPicker.newChooseAccountIntent(selectedAccount,
        null,
        new String[] {GoogleAccountManager.ACCOUNT_TYPE},
        true,
        null,
        null,
        null,
        null);`
so generally the same parameters so where is the fix?

Reply:
Hi @grzesiek2010! Thanks for reviewing. I could not find much in docs about the theme. But I was not able to change the theme by overriding in intent, in both cases. 
I agree that both [`AccountPicker.newChooseAccountIntent()`](https://developers.google.com/android/reference/com/google/android/gms/common/AccountPicker#newChooseAccountIntent(com.google.android.gms.common.AccountPicker.AccountChooserOptions)) and [`AccountManager.newChooseAccountIntent()`] take the same parameters, but the the default theme in both cases is different, so I decided to create intent in differnet ways, for light and dark theme.

Reply:
> but the the default theme in both cases is different

I can't see it on my android 5, I mean it's still dark always.

Reply:
I tried testing on the emulator. I found the same problem you mentioned on android 5. However, I did.t find it on Android 5.1, 6 and on my device running Android 9.

Reply:
We shouldn't merge such a fix if it doesn't wok on all devices and is kind of suspicious.

Reply:
Ok so I'm closing. @SaumiaSinghal thanks for trying!